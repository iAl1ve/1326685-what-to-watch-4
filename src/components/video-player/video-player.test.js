import React from "react";
import renderer from "react-test-renderer";
import MovieCard from "./video-player.jsx";
import ListMovies from "../../mock/testing.js";

describe(`Test Render MovieCard`, () => {
  it(`Render MovieCard`, () => {
    const tree = renderer
      .create(<MovieCard
        src = {ListMovies[0].src}
        preview = {ListMovies[0].preview}
      />, {
        createNodeMock: () => {
          return {};
        }
      }).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
