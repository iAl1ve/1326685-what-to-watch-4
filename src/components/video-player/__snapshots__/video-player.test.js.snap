// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Render MovieCard Render MovieCard 1`] = `
<video
  autoPlay={true}
  muted={true}
/>
`;
